### ISSUE TITLE
Amazon schickt Verifizierungscode. Keine Eingabemöglichkeit bei MoneyMoney

### ISSUE THREAD

**Issue by dvdwnrch** [Michael-Beutling/Amazon-MoneyMoney#69]
Hallo!
Bei Abfrage des Amazon Kontos wird mir per Mail ein 6-stelliger Verifizierungscode zugesendet.
Leider habe ich noch keine Möglichkeit gefunden diesen einzugeben.
Nach Ablauf des Zeitlimits zur Eingabe des Codes, bekomme ich mehrere Male erneut einen Code zugesendet.
VG
D

**Comment by Endereco** [comment]
ich kann das Problem bestätigen - Ein Counter läuft runter, E-Mail kommt auch. Allerdings gibts keine Möglichen 2FA abzusenden. Es gibt aber eine Rechnung API - die ggf. genutzt werden kann. (Invoicefetcher hat diese mit Amazon gebaut). vielleicht kann man diese als Basis nutzen.

**Comment by renkrich** [comment]
Kann ich auch bestätigen. Es funktioniert keine Umsatzabfrage.

**Comment by jsadbjsadljnsa** [comment]
Ja, leider seit mehreren Wochen. Keine Aussicht auf ein Fix von @Michael-Beutling.

**Comment by Uli007git** [comment]
> Ja, leider seit mehreren Wochen. Keine Aussicht auf ein Fix von @Michael-Beutling.

Bei mir funktioniert das immer noch nicht :( ... @jsadbjsadljnsa  bei Dir?

**Comment by jsadbjsadljnsa** [comment]
Mal geht es, mal nicht. 
Ich habe es zwar noch installiert, nutze es aber nicht mehr wirklich. Freue mich, wenn ab und an mal eine Bestellung dort angezeigt wird.

**Comment by aaronk6** [comment]
Ich konnte das Problem mit der Umstellung von SMS auf TOTP im Amazon-Account lösen. Ich wurde dann in MoneyMoney nach dem TOTP-Code gefragt, was einwandfrei funktioniert hat.

**Comment by Uli007git** [comment]
das hört sich gut an :) ich bekomme immer noch die SMS, was muss ich machen?

**Comment by aaronk6** [comment]
Schau mal unter _Ihr Konto › Anmelden und Sicherheit › Einstellungen für die Zwei-Schritt-Verifizierung (2SV) › Zwei-Schritt-Verifizierung_. Da bekommst du einen QR-Code angezeigt, um OTP aufzusetzen.

**Comment by Uli007git** [comment]
ok, habe ich gemacht aber er zieht sich nicht alle alten Daten, vielleicht lösch ich die Konten mal und lass ihn neu einlesen... die Aktion wäre vielleicht was für Weihnachten ;)

**Comment by Uli007git** [comment]
OTP habe ich eingerichtet, aber hat auch nicht viel genützt, obwohl ich die Amazon Accounts gelöscht und wieder neu mit OTP eingelesen habe. Hat aber nix genützt, die Daten wurden sogar schlechter und nur bis Juli eingelesen. Ich geb's auf glaube ich ... ich nutze es auch nicht wirklich

**Comment by dvdwnrch** [comment]
Bei MM die Zugangsdaten neu eingeben, dann den OTP Code.
Ältere Bestellungen werden nicht angezeigt, Einkäufe bei Amazon Video schon...
Ansonsten funktioniert es erstmal wieder. Vielen Dank!
